### PR TITLE
Fix missing description and http to https links

### PR DESCRIPTION
**Pull request by velkovb**

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/dhcpserver/Get-DhcpServerv4FreeIPAddress.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ ms.assetid: 83C2727D-5E10-4DA7-915B-476AF11A3F29
 # Get-DhcpServerv4FreeIPAddress
 
 ## SYNOPSIS
-Gets free IPv4 addresses from a specified scope.
+Gets one or more free, or unassigned, IPv4 Addresses from the specified scope.
```
Comment (JohanFreelancer9) [inline docset/windows/dhcpserver/Get-DhcpServerv4FreeIPAddress.md]:
```suggestion
Gets one or more free or unassigned IPv4 Addresses from the specified scope.
```

File: docset/windows/dhcpserver/Get-DhcpServerv4FreeIPAddress.md [hunk 2]
```diff
@@ -83,8 +83,8 @@ Use this parameter to run commands that take a long time to complete.
 The cmdlet immediately returns an object that represents the job and then displays the command prompt.
 You can continue to work in the session while the job completes.
 To manage the job, use the `*-Job` cmdlets.
-To get the job results, use the [Receive-Job](http://go.microsoft.com/fwlink/?LinkID=113372) cmdlet. 
-For more information about Windows PowerShell® background jobs, see [about_Jobs](http://go.microsoft.com/fwlink/?LinkID=113251).
+To get the job results, use the [Receive-Job](https://go.microsoft.com/fwlink/?LinkID=113372) cmdlet. 
+For more information about Windows PowerShell® background jobs, see [about_Jobs](https://go.microsoft.com/fwlink/?LinkID=113251).
```
Comment (JohanFreelancer9) [inline docset/windows/dhcpserver/Get-DhcpServerv4FreeIPAddress.md]:
```suggestion
For more information about Windows PowerShell background jobs, see [about_Jobs](https://go.microsoft.com/fwlink/?LinkID=113251).
```

File: docset/winserver2012-ps/dhcpserver/Get-DhcpServerv4FreeIPAddress.md [hunk 3]
```diff
@@ -25,15 +25,15 @@ Get-DhcpServerv4FreeIPAddress [-ScopeId] <IPAddress> [[-NumAddress] <UInt32>] [-
 ## DESCRIPTION
 This **Get-DhcpServerv4FreeIPAddress** cmdlet gets one or more free IP Addresses from the specified scope.
 
-If the **NumAddress** parameter is specified, then the requested number of free IPv4 addresses are returned.
-If the **NumAddress** parameter is not specified, then a single free IP address will be returned.
+If the *NumAddress* parameter is specified, then the requested number of free IPv4 addresses are returned.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/dhcpserver/Get-DhcpServerv4FreeIPAddress.md]:
```suggestion
If the *NumAddress* parameter is specified, then the requested number of free IPv4 addresses is returned.
```

File: docset/winserver2012-ps/dhcpserver/Get-DhcpServerv4FreeIPAddress.md [hunk 4]
```diff
@@ -69,7 +69,13 @@ This example gets a free IPv4 address from the specified scope within the IP add
...
 
 ### -AsJob
-ps_cimcommon_asjob
+Runs the cmdlet as a background job.
+Use this parameter to run commands that take a long time to complete. 
+The cmdlet immediately returns an object that represents the job and then displays the command prompt.
+You can continue to work in the session while the job completes.
+To manage the job, use the `*-Job` cmdlets.
+To get the job results, use the [Receive-Job](https://go.microsoft.com/fwlink/?LinkID=113372) cmdlet. 
+For more information about Windows PowerShell® background jobs, see [about_Jobs](https://go.microsoft.com/fwlink/?LinkID=113251).
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/dhcpserver/Get-DhcpServerv4FreeIPAddress.md]:
```suggestion
For more information about Windows PowerShell background jobs, see [about_Jobs](https://go.microsoft.com/fwlink/?LinkID=113251).
```

File: docset/winserver2012r2-ps/dhcpserver/Get-DhcpServerv4FreeIPAddress.md [hunk 5]
```diff
@@ -33,15 +33,15 @@ Get-DhcpServerv4FreeIPAddress [-ComputerName <String>] [-ScopeId] <IPAddress> [[
 ## DESCRIPTION
 This **Get-DhcpServerv4FreeIPAddress** cmdlet gets one or more free IP Addresses from the specified scope.
 
-If the **NumAddress** parameter is specified, then the requested number of free IPv4 addresses are returned.
-If the **NumAddress** parameter is not specified, then a single free IP address will be returned.
+If the *NumAddress* parameter is specified, then the requested number of free IPv4 addresses are returned.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/dhcpserver/Get-DhcpServerv4FreeIPAddress.md]:
```suggestion
If the *NumAddress* parameter is specified, then the requested number of free IPv4 addresses is returned.
```

File: docset/winserver2012r2-ps/dhcpserver/Get-DhcpServerv4FreeIPAddress.md [hunk 6]
```diff
@@ -77,7 +77,13 @@ This example gets a free IPv4 address from the specified scope within the IP add
...
 
 ### -AsJob
-ps_cimcommon_asjob
+Runs the cmdlet as a background job.
+Use this parameter to run commands that take a long time to complete. 
+The cmdlet immediately returns an object that represents the job and then displays the command prompt.
+You can continue to work in the session while the job completes.
+To manage the job, use the `*-Job` cmdlets.
+To get the job results, use the [Receive-Job](https://go.microsoft.com/fwlink/?LinkID=113372) cmdlet. 
+For more information about Windows PowerShell® background jobs, see [about_Jobs](https://go.microsoft.com/fwlink/?LinkID=113251).
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/dhcpserver/Get-DhcpServerv4FreeIPAddress.md]:
```suggestion
For more information about Windows PowerShell background jobs, see [about_Jobs](https://go.microsoft.com/fwlink/?LinkID=113251).
```

**Pull request merged**